docs/src/conf.py

```diff
@@ -503,7 +503,7 @@ def autolog(message: str, section: str | None = None, color: str | None = None)
 linkcheck_ignore = [
     "https://doi.org/10.5281/zenodo.7608302",
     "https://www.mtu.edu/geo/community/seismology/learn/earthquake-measure/magnitude/",
-    "region-manifold-extraction.html",
```

Comment:
We're now using a `ref` to the tutorial as a best practice, so we can remove this :+1: 

See this [PR#1146 comment](https://github.com/bjlittle/geovista/pull/1146/files#r1794286120).